kitchen.yml

```diff
@@ -26,18 +27,28 @@ suites:
   - name: default
     run_list:
       - recipe[test::install]
+      - recipe[test::sleep]
 
   - name: ldap
     run_list:
       - recipe[test::ldap]
+      - recipe[test::sleep]
 
   - name: proxy
     run_list:
       - recipe[test::proxy]
+    provisioner:
+      enforce_idempotency: false
+      multiple_converge: 1
```

Comment:
Is this resource not idempotent at the moment?

Reply:
The `grafana_user` update is not.

Reply:
@detjensrobert can you add a comment to the other suites that aren't (likely due to this?)